src/SFA.DAS.FAT.Web.UnitTests/Models/CoursesViewModelTests/WhenCreatingCoursesViewModel.cs

```diff
@@ -429,4 +429,76 @@ public void Then_ToQueryString_Should_Include_Routes_When_SelectedRoutes_Are_Set
 
         Assert.That(_sut.FindIndex(a => a.Item1 == "Categories"), Is.AtLeast(0));
     }
+
+    [Test]
+    public void When_Location_Is_Set_Then_Location_And_Distance_Must_Be_Added_To_Standard_Routes()
```

Comment:
We have deprecated this view model so I am not sure why we are adding unit tests here?

Reply:
Where these meant for new model?